common/common.go

```diff
@@ -13,3 +46,50 @@ func EqualSlices[P comparable](s1, s2 []P) bool {
 
 	return true
 }
+
+func ConvertToBytes(s string) (uint64, error) {
```

Comment:
Let's rename this to be better reflective of the fact that we're converting to digital units